src/test/ui-fulldeps/auxiliary/proc-macro-panic.rs

```diff
@@ -0,0 +1,13 @@
+// force-host
+// no-prefer-dynamic
+
+#![crate_type = "proc-macro"]
+
+extern crate proc_macro;
+use proc_macro::{TokenStream, Ident, Span};
+
+#[proc_macro]
+pub fn panic_in_libproc_macro(_: TokenStream) -> TokenStream {
```

Comment:
`test-macros.rs` has a panicking macro already.

Reply:
Ah, right, this is a fulldeps test.

Reply:
I thought that this test should go in `ui-fulldeps`, since it needs to be run with stage2.
Also, panicking inside a proc-macro doesn't trigger this issue - it requires the panic to originate from inside `libproc_macro` (or `rustc_expand/src/proc_macro_server.rs`).